src/linear_solver_logs/solve_log_full_ma.jl

```diff
@@ -0,0 +1,179 @@
+# This file is part of RLinearAlgebra.jl
+
+# Using the structs, MAInfo, in the file that implementing MA. 
+include("solve_log_ma.jl")
+
+"""
+    LSLogFullMA <: LinSysSolverLog
+
+A mutable structure that stores information about a randomized linear solver's behavior.
+    The log assumes that the full linear system is available for processing. The goal of
+    this log is usually for research, development or testing as it is unlikely that the
+    entire residual vector is readily available.
+
+# Fields
+- `collection_rate::Int64`, the frequency with which to record information to append to the
+    remaining fields, starting with the initialization (i.e., iteration 0).
+- `resid_hist::Vector{Float64}`, retains a vector of numbers corresponding to the residual
+    (uses the whole system to compute the residual)
```

Comment:
Needs to be clarified that this the norm of the residuals at the collection iterates